src/usr/local/www/firewall_rules_edit.php

```diff
@@ -1216,6 +1258,7 @@ function build_if_list() {
 	'Protocol',
 	$pconfig['proto'],
 	array(
+		'any' => gettext('any'),
```

Comment:
This change is unwarranted. The protocol default should remain TCP, it is set that way on purpose (otherwise people get confused by the lack of port options being visible). It's also not relevant to the other changes being made on this PR.